biojava-genome/src/test/java/org/biojava/nbio/genome/TestChromosomeMappingTools.java

```diff
@@ -0,0 +1,61 @@
+package org.biojava.nbio.genome;
+
+import org.biojava.nbio.genome.util.ChromosomeMappingTools;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static junitx.framework.ComparableAssert.assertEquals;
```

Comment:
This should be imported from package org.junit.assert. I usually simply do this:
```java
import static org.junit.Assert.*;
```
which imports all the assert static methods.